server/app/src/androidTest/java/sh/calaba/instrumentationbackend/entrypoint/AndroidInstrumentationStartup.java

```diff
@@ -180,4 +188,19 @@ private String detectMainActivity(StatusReporter statusReporter, String targetPa
             throw new RuntimeException(e);
         }
     }
+
+    private void reportFailure(String message) {
+        hasReportedFailure = true;
+
+        if (isSdk11()) {
+            InstrumentationReport.send(instrumentation, DISPLAY_NAME, FAILED, message);
+        } else {
+            StatusReporter statusReporter = new StatusReporter(instrumentation.getContext());
+            statusReporter.reportFailure(message);
+        }
+    }
+
+    private boolean isSdk11() {
```

Comment:
Please rename it: `isGreaterThanSDK11` or something like that.

Reply:
I will